.vitepress/theme/styles/fonts.css

```
@@ -77,16 +77,16 @@
 
 /* Preload critical fonts */
 @media screen {
-  /* Preconnect to Google Fonts */
-  link[rel="preconnect"][href*="fonts.googleapis.com"],
-  link[rel="preconnect"][href*="fonts.gstatic.com"] {
-    crossorigin: anonymous;
-  }
+    /* Preconnect to Google Fonts */
+    link[rel="preconnect"][href*="fonts.googleapis.com"],
+    link[rel="preconnect"][href*="fonts.gstatic.com"] {
+        crossorigin: anonymous;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)